mock-echo-app/src/cmr/mock_echo/client/echo_util.clj

```diff
@@ -478,3 +487,22 @@
          :provider_identity
          {:target "GROUP"
           :provider_id provider-guid}))
+
+(defn grant-permitted?
```

Comment:
Cool! I feel a lot better now. In that case, the other functions may be moved as well, but probably all to a better place. I will let you handle it.